docs/user_guide/source/api_full/fortran.rst

```diff
@@ -120,15 +120,15 @@ ADIOS2 Fortran bindings handlers are mapped 1-to-1 to the ADIOS components descr
       logical, intent(in):: adios2_debug_mode
       
 
-* :f90:`subroutine adios2_declare_io` spawn io components
+* :f90:`subroutine adios2_declare_io` spawn/create an io component
 
    .. code-block:: fortran
 
       subroutine adios2_declare_io(io, adios, io_name, ierr)
       
       ! WHERE:
       
-      ! io component that defines an IO tasks inside adios component
```

Comment:
Add error code, for completeness sake :)
```
! error code
integer, intent(out) :: ierr
```
Also needed for:
1. adios2_init
2. adios2_at_io
3. adios2_flush_all
4.  adios2_finalize 
5. adios2_set_engine 
6. adios2_set_parameter 
7. adios2_flush_all_engines 
8. adios2_remove_variable 
9. adios2_remove_attribute 
10. adios2_remove_all_attributes
11. adios2_define_variable
12. adios2_inquire_variable
13. adios2_open
14. adios2_begin_step
15. adios2_current_step
16. adios2_steps
17. adios2_end_step
18. adios2_put
19. adios2_perform_puts
20. adios2_get
21. adios2_perform_gets
22. adios2_close
23. adios2_define_attribute 
24. adios2_inquire_attribute